domain/domain.go

```diff
@@ -817,7 +817,16 @@ func (do *Domain) Init(ddlLease time.Duration, sysExecutorFactory func(*Domain)
 	if err != nil {
 		return err
 	}
-	do.globalCfgSyncer = globalconfigsync.NewGlobalConfigSyncer(do.etcdClient)
+	// try cast do.store to kv.Storage
```

Comment:
? `do.store` is already `kv.Storage`, no need cast.

Reply:
no it does not contain such method.